packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -94,11 +94,7 @@ function BlockListAppender( {
 			// Prevent the block from being selected when the appender is
 			// clicked.
 			onFocus={ stopPropagation }
-			className={ classnames(
-				'block-list-appender',
-				'wp-block',
-				className
-			) }
+			className={ classnames( 'block-list-appender', className ) }
```

Comment:
`wp-block` was added here in https://github.com/WordPress/gutenberg/pull/25849, but I don't think it should be. Everything still works without it.

Reply:
Yeah, it should be a mistake 😓

Reply:
Why shouldn't it be there? Won't it give correct margins to the appender?

Reply:
Didn't seem to make a difference from what I can tell.

Reply:
It does make a difference, we should restore it, without it, the block appender won't be considered a "block" meaning it won't be aligned properly in a block list.

For instance you can test this by adding an image block and notice that the appender that shows after the image is not centered properly.